proto/buf/alpha/registry/v1alpha1/admin.proto

```diff
@@ -259,8 +277,27 @@ message GetClusterUsageRequest {
   google.protobuf.Timestamp start_time = 1;
   // The timestamp of the ending time of a date range.
   google.protobuf.Timestamp end_time = 2;
+  // Entity id is a optional owner or repository id that we want to
+  // display usage for.
+  oneof entity_id {
+    // The id of the owner.
+    string owner_id = 3;
+    // The id of the repository.
+    string repository_id = 4;
+  }
 }
 
 message GetClusterUsageResponse {
   repeated MonthlyUsage usage = 1;
 }
+
+message GetAllOrganizationsUsagesRequest {
+  // The timestamp of the beginning time of a date range.
+  google.protobuf.Timestamp start_time = 1;
+  // The timestamp of the ending time of a date range.
+  google.protobuf.Timestamp end_time = 2;
+}
+
+message GetAllOrganizationsUsagesResponse {
+  repeated RepositoryUsage repository_usages = 1;
```

Comment:
What is repeated here? What is unique and what is not? All needs to be documented

Reply:
// The list of types usage for all repositories in all orgs on a single BSR instance.

Reply:
This doesn't answer the original question - is this monthly? What does the list contain? What should I expect a valid list to include based on the timestamp?

Reply:
`// The list contains types usage info and counts for all repositories in all orgs on a single BSR instance, based on a snapshot single point in time within the requested time range where the aggregate types usage is at a maximum.`

does this make more sense?


> is this monthly? 

no, it's for any point within the requested date range

> What does the list contain? 

repository information and types usage for all repositories on the instance

> What should I expect a valid list to include based on the timestamp?

the types count for the point when types usage was at a maximum in the time range

Reply:
Between the documentation of this method, and the documentation/name for RepositoryUsage, some things are very unclear
* How many items will be in this list? Do we need to have paging?
* What is the timeframe associated with each item? Is this configurable? Should it be?
* How do I map items in this list to a particular timeframe?

Other thought:
* If this method is really specialized toward returning max usage within some time horizon, maybe the names should more explicitly represent that.